app/src/androidTest/java/org/mozilla/fenix/ui/TabbedBrowsingTest.kt

```diff
@@ -219,6 +220,7 @@ class TabbedBrowsingTest {
     }
 
     @Test
+    @Ignore("Intermittent test")
```

Comment:
This is not an intermittent but seems to actually crash:

```
java.lang.IllegalArgumentException
	at androidx.core.text.TextDirectionHeuristicsCompat$TextDirectionHeuristicImpl.isRtl(TextDirectionHeuristicsCompat.java:131)
	at mozilla.components.support.utils.URLStringUtils.toDisplayUrl(URLStringUtils.kt:110)
	at mozilla.components.support.utils.URLStringUtils.toDisplayUrl$default(URLStringUtils.kt:104)
	at org.mozilla.fenix.components.toolbar.BrowserToolbarView$2$1$2.invoke(BrowserToolbarView.kt:121)
	at org.mozilla.fenix.components.toolbar.BrowserToolbarView$2$1$2.invoke(BrowserToolbarView.kt:121)
	at mozilla.components.browser.toolbar.display.DisplayToolbar.setUrl$browser_toolbar_release(DisplayToolbar.kt:485)
	at mozilla.components.browser.toolbar.BrowserToolbar.setUrl(BrowserToolbar.kt:110)
	at mozilla.components.feature.toolbar.internal.URLRenderer.updateUrl$feature_toolbar_release(URLRenderer.kt:67)
	at mozilla.components.feature.toolbar.internal.URLRenderer$start$1.invokeSuspend(URLRenderer.kt:41)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
```

Reply:
So we shouldn't land it but fix the root cause.

Reply:
@Mugurell this could be related to 
https://github.com/mozilla-mobile/android-components/commit/b1ddb39f78917fce7f14cc5cc184868d052cf383

Reply:
More details in [Firebase](https://console.firebase.google.com/u/1/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/8906963029425277909/executions/bs.c2f7edc2f22dc798/issues).

Reply:
Looks like with the current params (start 0 and count 1) this will happen if the text passed to `isRtl` is empty.

Reply:
Thank you all and sorry for the troubles.
Pushed an updated to AC  - https://github.com/mozilla-mobile/android-components/pull/10640

Reply:
Thanks for the fix!